<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomWindow</class>
 <widget class="QDialog" name="CustomWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Удаление игры</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    border-radius: 10px;
}
QLabel {
    font-size: 12px;
}
QLabel#gameNameLabel {
    color: #4CAF50;
    font-size: 14px;
    font-weight: bold;
    padding: 8px;
    border-radius: 5px;
}
QPushButton {
    background-color: #4CAF50;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 8px 15px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #45a049;
}
QPushButton:pressed {
    background-color: #3d8b40;
}
QPushButton#deleteButton {
    background-color: #f44336;
}
QPushButton#deleteButton:hover {
    background-color: #da190b;
}
QPushButton#deleteButton:pressed {
    background-color: #b71c1c;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="gameNameLabel">
     <property name="text">
      <string>Удалить игру {Игра}?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
